src/state.rs

```diff
@@ -265,6 +243,36 @@ impl<'a, S: Storage> Config<'a, S> {
         self.storage
             .set(KEY_CONTRACT_STATUS, &status_u8.to_be_bytes());
     }
+
+    pub fn set_minters(&mut self, minters_to_set: Vec<HumanAddr>) -> StdResult<()> {
+        set_bin_data(&mut self.storage, KEY_MINTERS, &minters_to_set)?;
+
+        Ok(())
+    }
+
+    pub fn add_minters(&mut self, minters_to_add: Vec<HumanAddr>) -> StdResult<()> {
+        let mut minters = self.minters();
+        minters.extend(minters_to_add);
+
+        self.set_minters(minters)
```

Comment:
This can create duplicates if the new list of minters has addresses that were listed previously.
You can convert to a `HashMap` or `BtreeMap`, then add the items, then `.iter().collect()` back into a `Vec`

Reply:
I thought about using `HashSet` instead of `Vec` in general, but it seemed kinda expensive in some use cases.
Isn't `.iter().collect()` too expensive with regard to the risk of having the same minter appear twice? (remember - this is an admin command)

In any case, I'm thinking about just dropping `AddMinters` and `RemoveMinters` and keep just `SetMinters`. Tell me what you think @reuvenpo

Reply:
I think i'd keep `AddMinters` and `RemoveMinters`. Perhaps a cheaper solution is to (In `RemoveMinters`) continue `remove()`ing from the the list as long as it `.contain()`s the item you're trying to remove.
```rust
while minters.contains(&minter) {
  minters.remove(&minter);
}
```